smaps/src/test/java/com/google/smaps/HomepageTest.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.smaps;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+/**
+ * Unit tests for {@link Homepage}.
+ */
+
+@RunWith(JUnit4.class)
+public class HomepageTest {
+  private static final String FAKE_URL = "fake.fk/home";
+  // Set up a helper so that the ApiProxy returns a valid environment for local testing.
+  private final LocalServiceTestHelper helper = new LocalServiceTestHelper();
+
+  @Mock private HttpServletRequest mockRequest;
+  @Mock private HttpServletResponse mockResponse;
+  private StringWriter responseWriter;
+  private Homepage servletUnderTest;
+
+  @Before
+  public void setUp() throws Exception {
+    MockitoAnnotations.initMocks(this);
+    helper.setUp();
+
+    //  Set up some fake HTTP requests
+    when(mockRequest.getRequestURI()).thenReturn(FAKE_URL);
+
+    // Set up a fake HTTP response.
+    responseWriter = new StringWriter();
+    when(mockResponse.getWriter()).thenReturn(new PrintWriter(responseWriter));
+
+    servletUnderTest = new Homepage();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void doGet_writesResponse() throws Exception {
+    servletUnderTest.doGet(mockRequest, mockResponse);
+
+    // We expect our homepage response.
+    assertThat(responseWriter.toString()).named("Homepage response").contains("Home - Standard ");
+  }
+
+  @Test
+  public void HomepageInfo_test() {
```

Comment:
lowercase homepage, as per https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names.
Let's also come up with a consistent naming convention for tests. I think "_test" is redundant, so maybe just drop and name this test "homepageInfo" until it does more where we can add a "_state."